compiler/rustc_abi/src/lib.rs

```diff
@@ -1463,16 +1467,18 @@ impl BackendRepr {
         matches!(*self, BackendRepr::Scalar(s) if s.is_bool())
     }
 
-    /// Returns the fixed alignment of this ABI, if any is mandated.
-    pub fn inherent_align<C: HasDataLayout>(&self, cx: &C) -> Option<AbiAndPrefAlign> {
-        Some(match *self {
-            BackendRepr::Scalar(s) => s.align(cx),
-            BackendRepr::ScalarPair(s1, s2) => s1.align(cx).max(s2.align(cx)),
+    /// fixed alignment of this repr, if any is mandated
```

Comment:
This seems to say the same thing as the old sentence, just without being a properly capitalized sentence?

Reply:
And the trailing dot as well. ;)